### ISSUE TITLE
Add a link to the memories app

### ISSUE THREAD

**Issue by philippgmk** [nextcloud/photos#1509]
Hello.

I have used nextcloud on and off for a long time and photos continue to be my primary reason for looking else where. I don't mean to be disrespectful, but it can be acknowledged that the default photos app is not usable if you have a lot of photos. There has been a lot of discussion about this on this on reddit, but I am not sure the devs use reddit, so posting this here. From a google search,
* https://www.reddit.com/r/NextCloud/comments/n5jaqz/nextcloud_photos_still_useless/
* https://www.reddit.com/r/NextCloud/comments/k6qmt8/alternative_to_nextclouds_photo_gallery/
* https://www.reddit.com/r/NextCloud/comments/jue32m/viewing_nextcloud_photos_without_losing_ones_mind/
* https://www.reddit.com/r/NextCloud/comments/qr9q4d/nextcloud_app_for_better_photo_sharing/

I only very recently came to know about this app called memories from another reddit post https://www.reddit.com/r/selfhosted/comments/z245mo/loading_40k_images_in_one_view_with_memories/. I have been using it for a few days and it is very smooth and usable for thousands of photos, and as good as even dedicated photo apps like photoprism. But unfortunately I never found it before because there is no way unless you search for it or look through all apps in the app store. Link to the app: https://github.com/pulsejet/memories

The creator of memories has mentioned there are no plans to make the app official https://github.com/pulsejet/memories/issues/183, but would it be possible to at least add a link somewhere to let users know that something like this exits? Again I don't mean to be disrespectful, but I think the lack of a good photos app for large libraries hurts nextcloud, especially when so many of us run a server especially for photo backup. And the only solution you see for nextcloud when you first install nextcloud is the default photos app.

P.S. I don't use GitHub (this is only my second post!) so please close and let me know if this is not appropriate here.

**Is your feature request related to a problem? Please describe.**
No link to the memories app

**Describe the solution you'd like**
Add a link

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by meichthys** [comment]
@philippgmk I agree fully with your sentiments, but i think we'll need to request the featured tag in the [AppStore Repository](https://github.com/nextcloud/appstore).

I've submitted a request there with a reference to this issue.

**Comment by joshtrichards** [comment]
There's nothing wrong with this request per se, but I don't think this is the appropriate channel for it.

I suggest this be closed (here in the Photos repo) since it's a distraction - at best - to the Photos devs/contributors (none are in a position to address the request so this will just sit here open).

I made two suggestions on the Issue created by @meichthys in the appstore repo as to how you might consider proceeding. 

But this discussion doesn't belong here - in this repo.

P.S. I'm an outside/community contributor to other NC repos, so don't take this as anything official.
